project/ws/app/src/lib/routes/home/components/user-cards/user-card.component.html

```diff
@@ -48,187 +41,240 @@
+                    i18n-aria-label aria-label="Please enter a valid 10 digit mobile number">
+                    Please enter a valid 10 digit mobile number
+                  </mat-error>
+                </mat-form-field>
+                <!-- <div class="flex m-auto" *ngIf="isMobileVerified">
+                  <mat-icon class="ws-mat-green-text margin-xs" svgIcon="verified"></mat-icon>
+                </div> -->
+                <!-- <div *ngIf="!isMobileVerified" class="flex flex-1">
+                  <button *ngIf="!otpSend" mat-raised-button type="button" (click)="sendOtp()"
+                    class="text-white btn-success btn-responsive" [disabled]="updateUserDataForm?.mobile?.invalid">
+                    <a class="text-white" i18n>
+                      Send OTP
+                    </a>
+                  </button>
+                  <div *ngIf="otpSend" class="flex flex-1 flex-column">
+                    <div class="flex flex-1 flex-row margin-bottom-m">
+                      <mat-form-field appearance="outline" class="margin-top-xs flex-expand w-minus-w50">
+                        <input id="mobileNumber" #otp type="number" matInput placeholder="Enter OTP" i18n-aria-label
+                          aria-label="OTP input" onKeyPress="if(this.value.length === 8) return false;">
+                        <mat-hint *ngIf="timeLeftforOTP > 0 ">
+                          <span>Resend OTP after {{timeLeftforOTP | pipeDurationTransform:
+                            'hms2M'}}</span>
+                        </mat-hint>
+                        <mat-hint *ngIf="timeLeftforOTP <= 0 ">
+                          <span class="cursor-pointer ws-mat-default-text" (click)="resendOTP()">
+                            Resend OTP
+                          </span>
+                        </mat-hint>
+                      </mat-form-field>
+                      <button mat-raised-button type="button" (click)="verifyOtp(otp)"
+                        class="text-white btn-success btn-responsive margin-top-s otp"
+                        [disabled]="updateUserDataForm?.mobile?.invalid  || disableVerifyBtn">
+                        <span class="text-white" i18n>
+                          Verify OTP
+                        </span>
+                      </button>
+                    </div>
                   </div>
-                </mat-card>
-              </mat-grid-tile>
+                </div> -->
+              </div>
+            </div>
+            <div class="flex flex-row margin-top-m">
+              <div class="md:w-1/3">
+                <label class="margin-remove-bottom form-label required">Date of Birth</label>
+                <mat-form-field appearance="outline" class="customdob">
+                  <input id="birthDate" matInput #dob formControlName="dob" [matDatepicker]="picker" i18n-placeholder
+                    (focus)="picker.open()" placeholder="Choose date of birth" [min]="1/1/1900" [max]="today">
```

Comment:
please check for min date with product

Reply:
Have added same value as in  learner portal edit profile component